ecs-cli/modules/cli/compose/entity/service/service_test.go

```diff
@@ -809,10 +867,36 @@ func TestUpdateExistingServiceWithDesiredCountOverOne(t *testing.T) {
 	// define expected client input given the above info
 	expectedInput := getDefaultUpdateInput()
 	expectedInput.serviceName = serviceName
-	expectedInput.count = *aws.Int64(int64(existingDesiredCount))
+	expectedInput.count = aws.Int64(int64(existingDesiredCount))
+
+	// call tests
+	updateServiceTest(t, flagSet, &config.CommandConfig{}, &utils.ECSParams{}, expectedInput, existingService)
+}
+
+func TestUpdateExistingServiceWithDaemonSchedulingStrategy(t *testing.T) {
```

Comment:
can you add a test that asserts an error is thrown if `updateService` is attempted with the `--scheduling-strategy` flag set?

Reply:
Added in https://github.com/aws/amazon-ecs-cli/pull/685/commits/07aba4f8acd4665eb17d4c7dca976fafa8efe962.